packages/user/Tokens/service/src/lib.rs

```diff
@@ -461,7 +461,7 @@ pub mod service {
     /// * `memo`     - Memo
     #[action]
     fn reject(token_id: TID, creditor: AccountNumber, memo: Memo) {
-        SharedBalance::get_assert(creditor, get_sender(), token_id).reject(memo);
+        SharedBalance::get_or_new(creditor, get_sender(), token_id).reject(memo);
```

Comment:
This changes `reject` from a fail if invalid to a no-op.

1) If we are going to use no-op, this should just conditionally call reject if get(...).is_some(), instead of creating an empty SharedBalance and immediately rejecting it.
2) Should it be a no-op? `credit` and `debit` fail if the amount is 0. Reject seems to me like the most likely action where a no-op is justified... but it does make it different than the other actions.

One other design to consider is a separate `reject_any` action that no-ops, and leave the default `reject` as an assert. Then the caller can express their intended usage based on which action is called.

Reply:
1. As disccused, we'll ignore that feedback and leave as is. 
2. Concluded in discussion, we're happy to no-op.